lib/Terminal/Print/RawInput.pm6

```diff
@@ -11,11 +11,11 @@ sub raw-input-supply(IO::Handle $input = $*IN,
                      Bool :$drop-old-unread) is export {
     # If a TTY, convert to raw mode, saving current mode first
     my $fd = $input.native-descriptor;
-    my $saved-termios;
+    my $saved-term-config;
     if $input.t {
-        $saved-termios = Terminal::MakeRaw::getattr($fd);
-        my $mode       = $drop-old-unread ?? :FLUSH !! :DRAIN;
-        Terminal::MakeRaw::makeraw($fd, |$mode);
+        $saved-term-config = Terminal::API::get-config($fd);
+        my $when           = $drop-old-unread ?? Terminal::API::FLUSH !! Terminal::API::DRAIN;
+        Terminal::API::make-raw($fd, :$when);
```

Comment:
`:when` is a somewhat odd name for this parameter.  Maybe `:how` or `:method` or `:cleanup` or so?

Reply:
This was inspired by the man page (highlight mine):

```
optional_actions specifies **when** the changes take effect:
TCSANOW
    the change occurs immediately. 
TCSADRAIN
    the change occurs after all output written to fd has been transmitted. This function should be used when changing parameters that affect output. 
TCSAFLUSH
    the change occurs after all output written to the object referred by fd has been transmitted, and all input that has been received but not read will be discarded before the change is made. 
```

Do you still think the name is suboptimal? (I don't have strong emotions, so if you think something else is better, I'll gladly change it.)

Reply:
Hmmm.  In that case, I think your choice is correct (to match the underlying documentation, since this is a relatively thin layer in the Unix world at least).

Whether the original API had good naming sense is a different question -- but not your problem to solve.  ;-)